components/layout/FavoriteActivityCard.js

```diff
@@ -0,0 +1,102 @@
+import { activities } from "@/lib/activities";
+import Image from "next/image";
+import { FaHeart } from "react-icons/fa";
+import styled from "styled-components";
+
+export default function FavoriteActivityCard({
+  id,
+  title,
+  country,
+  categories,
+  imageUrl,
+  toggleBookmark,
+  isBookmarked,
+}) {
+  return (
+    <StyledFavoriteCard>
+      <StyledImage src={imageUrl} alt={title} sizes="50vw" fill />
+      <CardHeader>
+        <div>
+          {categories.map((category) => (
+            <StyledTag key={category}>{category}</StyledTag>
```

Comment:
I would suggest to use id as a key.

Reply:
the categories don't have an id,  Felix said we should use category as key